src/components/core/geoChart/tests/GeoChartLegendRenderer.spec.tsx

```diff
@@ -78,4 +81,43 @@ describe("GeoChartLegendRenderer", () => {
         expect(await wrapper.find(".color-legend")).toHaveLength(1);
         expect(await wrapper.find(".s-pushpin-size-legend")).toHaveLength(1);
     });
+
+    it("should render component with PushpinSizeLegend and PushpinCategoryLegend", () => {
+        const execution = {
+            executionResponse: getExecutionResponse(true, true, false, true),
+        };
+        const config = getGeoConfig({ isWithLocation: true, isWithSize: true, isWithSegment: true });
+        const { mdObject: { buckets = [] } = {} } = config;
+        const geoData = getGeoData(buckets, execution.executionResponse.dimensions);
+
+        const props: IGeoChartLegendRendererProps = {
+            config,
+            geoData,
+            sizeData: [],
+            segmentData: [],
+        };
+        const wrapper = createComponent(props);
+        const sizeLegend = wrapper.find(PushpinSizeLegend);
+        const categoryLegend = wrapper.find(PushpinCategoryLegend);
+        expect(wrapper.find(".s-geo-legend")).toHaveLength(1);
+        expect(wrapper.find(".s-geo-category-legend")).toHaveLength(1);
+        expect(sizeLegend.length).toEqual(1);
+        expect(categoryLegend.length).toEqual(1);
+    });
+
+    it("should not render component ", () => {
```

Comment:
Do you mean `should not render Legend component`?
And why do we have 2 "should not render component"?
And please help me refactor test names to
```
"should not render component" -> "should render nothing"
"should render component with PushpinSizeLegend" -> "should render only Size legend"
"should render component with color legend" -> "should render only Color legend "
"should render component with size and color legend" -> "should render Size and Color legend "
"should render component with PushpinSizeLegend and PushpinCategoryLegend" -> "should render Size and Category legend "
"should not render component " -> "should render nothing with only Location bucket item"
```

Reply:
fixed